hedera-mirror-test/k6/src/web3/test/common.js

```diff
@@ -0,0 +1,39 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+const errorField = "error";
+const resultField = "result";
+
+function isNonErrorResponse(response) {
+  //instead of doing multiple type checks,
```

Comment:
nit: spacing on comments here and other places
```suggestion
  // instead of doing multiple type checks,
```

Reply:
I was told in another comment to use the auto-format from the idea. Maybe we can talk someday about having an IDE settings file.